src/solidity-frontend/solidity_convert.h

```diff
@@ -7,6 +7,7 @@
 #include <util/context.h>
 #include <util/namespace.h>
 #include <util/std_types.h>
+#include <clang-c-frontend/padding.h>
```

Comment:
Do we need this header?

Reply:
The `add_padding` in `padding.h` function is used to deal with the alignment in struct or union (i.e. bitfield). It occurs in the `clang_c_convert.cpp`.

I'm not quite sure if the `struct` and `union` need to be packed and aligned as in C++, but I would say I shouldn't include this "padding" in this commit. Thanks for the suggestion.